src/pages/about-us.astro

```diff
@@ -37,15 +37,10 @@ import { waitlistUrl } from '../waitlist';
 
       <div class="intro">
         <div class="text">
+          <p>This is us: Simen and Martin. Two builders.</p>
```

Comment:
I think this phrasing made sense when we had 3 people and 2 specialisations, but maybe this fits better now?:

```suggestion
          <p>Simen and Martin: Two builders.</p>
```